src/util/cached-prop-defs.js

```diff
@@ -0,0 +1,83 @@
+//@flow
+import assign from './assign';
+import dashCase from './dash-case';
+import getAllKeys from './get-all-keys';
+
+const CACHED_PROP_DEFS:string = '____skate_propDefs';
```

Comment:
I'd recommend pursuing TypeScript instead of Flow, one of the reasons being these kind of type inferences are made automatically.